### PR TITLE
List organizer organizations 

### PR DESCRIPTION
**Pull request by frinyvonnick**
Related #164

### CODE REVIEW

File: src/store/listeners/reactions/organizations.js [hunk 1]
```diff
@@ -28,3 +30,31 @@ export const createOrganization = reaction(async (action, store, { form }) => {
...
+        store.data.organizations.add(organization)
+      }
+
+      return organization
+    },
+  ))
+
+  store.ui.organizer.myOrganizations.reset()
+  store.ui.organizer.myOrganizations.set(organizerOrganizations)
+})
```
Comment (bpetetot) [inline src/store/listeners/reactions/organizations.js:57]: nice 👍

File: src/store/listeners/reactions/organizations.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import { reaction } from 'k-ramel'
 import { push } from 'redux-little-router'
+import { map } from 'lodash'
```
Comment (bpetetot) [inline src/store/listeners/reactions/organizations.js]: Can you do `import map from 'lodash/map'` ? 
With create-react-app I don't have lodash-plugin so tree-shaking is not so good.

File: src/store/listeners/reactions/organizations.js [hunk 3]
```diff
@@ -28,3 +30,31 @@ export const createOrganization = reaction(async (action, store, { form }) => {
   store.dispatch(push('/organizer/organizations'))
```
Comment (bpetetot) [inline src/store/listeners/reactions/organizations.js:27]: You have to add the new organization into the store in order to have it when you go to the route `'/organizer/organizations'`

**Review by bpetetot — CHANGES_REQUESTED** [review]

File: src/store/listeners/reactions/organizations.js (hunk 3 above)
Comment (frinyvonnick) [reply src/store/listeners/reactions/organizations.js:27]: Done in `refactoring/batch-organization-creation`

**Review by bpetetot — APPROVED** [review]

**Pull request merged**